src/createPrismicLink.ts

```diff
@@ -0,0 +1,104 @@
+import type { ApolloLink, HttpOptions } from "@apollo/client/core";
+import { FetchLike, getRepositoryName } from "@prismicio/client";
+import { createHttpLink } from "@apollo/client/core";
+import {
+	getRepositoryEndpoint,
+	getGraphQLEndpoint,
+	createClient,
+} from "@prismicio/client";
+
+/**
+ * Configuration for `createPrismicLink`.
+ */
+export type PrismicLinkConfig = Omit<
+	HttpOptions,
+	"fetch" | "useGETForQueries"
+> & {
+	/**
+	 * The name of the link's Prismic repository.
+	 */
+	repositoryName: string;
```

Comment:
`repositoryName` could be optional, because it is derived from `uri` if that is present

Reply:
Thanks for catching that. I've updated the type to support at least either `repositoryName` or `uri`.